README.md

```diff
@@ -0,0 +1,152 @@
+# Supported LXC Containers, by Symas Corp.
+
+## What's here?
+
+[OpenLDAP](https://openldap.org) and related software, provided by
+[Symas](https://symas.com), containerized and ready for production use.
+
+
+## How to contact Symas for commercial support
+
+If you've not tried running your container with `SYMAS_DEBUG` set to `true` then you might try that first, this container is primed to give you the information you need to debug issues yourself.  Also, please take a look at our [knowledge base](https://kb.symas.com/) if going to the [OpenLDAP site](https://openldap.org/) and reading the [documentation](https://openldap.org/doc/), the [quick start guide](https://openldap.org/doc/admin26/quickstart.html), and the detailed [manual pages](https://openldap.org/software/man.cgi) didn't help.  [What we publish](https://repo.symas.com) is what we provide inside the [containers](https://github.com/symas/containers) we provide to you, everything is open-source.
```

Comment:
The sentence structure could be improved.

Reply:
Agreed, updated it.